backend/zf/settings.py

```diff
@@ -29,6 +29,7 @@
 
 ALLOWED_HOSTS = [
     'localhost',
+    '192.168.99.100',
```

Comment:
Ideally we would externalize this somehow, but this is fine for now, because the default IP for docker machine is always 192.168.99.100.